cli/flags.rs

```diff
@@ -1059,6 +1059,7 @@ fn test_subcommand<'a, 'b>() -> App<'a, 'b> {
       Arg::with_name("jobs")
         .short("j")
         .long("jobs")
+        .help("Number of parallel jobs, defaults to # of CPUs when no value is provided")
```

Comment:
Is there a way to describe this without using "jobs" in order to explain what "jobs" means?

Reply:
Hmm for self explanatory, I guess worker would be a good approximation.

```suggestion
        .help("Number of parallel workers, defaults to # of CPUs when no value is provided")
```

Reply:
Yeah, I think that's better. It might be nice to somehow explain what the workers do as users might not know. I'm not sure what it should say myself as I'm not familiar with this feature.

Reply:
>  Ex. they're parallel across modules, right?

No, modules *can* run in parallel but they're still isolated to their isolate.